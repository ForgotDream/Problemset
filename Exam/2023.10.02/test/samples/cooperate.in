10
131176248 3
859484431 9
608413794 7
734575217 4
149798325 0
129566432 6
412776101 0
911759975 3
137806872 0
982907006 1
