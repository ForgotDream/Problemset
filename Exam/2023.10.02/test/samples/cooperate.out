764894493
854098542
884197521
654078735
0
844197552
0
884191176
0
702900000
